service/http.directory/src/test/scripts/org/kaazing/gateway/service/http/directory/get.file.directory.rpt

```diff
@@ -0,0 +1,9 @@
+connect http://localhost:8700/index.html
+connected
+
+write method "GET"
+
+read status "200" "OK"
+
+close
+disconnected
```

Comment:
disconnected should not be needed in this script

Nit: We try to end scripts in a newline